<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="FOCUS-QA-10" name="com.focus.testcases.SellerPriceBookTest" tests="31" failures="3" timestamp="30 Dec 2022 15:39:00 GMT" time="810.039" errors="1">
  <testcase name="checkCreatingBuyerPriceBookwithMaxandMinQuantity" time="36.352" classname="com.focus.testcases.SellerPriceBookTest"/>
  <testcase name="checkDeleteBuyerPriceBook" time="29.661" classname="com.focus.testcases.SellerPriceBookTest"/>
  <testcase name="checkCreatingBuyerPriceBookwithVendorandRate" time="31.526" classname="com.focus.testcases.SellerPriceBookTest"/>
  <testcase name="checkCreatingBuyerPriceBookforMinandMaxQuantity" time="22.256" classname="com.focus.testcases.SellerPriceBookTest"/>
  <testcase name="checkPurchaseVoucherRateFieldforVendorAccount" time="22.008" classname="com.focus.testcases.SellerPriceBookTest"/>
  <testcase name="checkValidatingtheRateFieldatPurchaseVoucher" time="33.511" classname="com.focus.testcases.SellerPriceBookTest"/>
  <testcase name="checkSavingSellerPriceBook" time="16.171" classname="com.focus.testcases.SellerPriceBookTest"/>
  <testcase name="checkDeletetheBuyerPriceBook" time="21.784" classname="com.focus.testcases.SellerPriceBookTest"/>
  <testcase name="checkVerifyingPurchaseVoucherforRateQuantityApplied" time="45.510" classname="com.focus.testcases.SellerPriceBookTest">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
	at org.testng.Assert.fail(Assert.java:94)
	at org.testng.Assert.failNotEquals(Assert.java:513)
	at org.testng.Assert.assertEqualsImpl(Assert.java:135)
	at org.testng.Assert.assertEquals(Assert.java:116)
	at org.testng.Assert.assertEquals(Assert.java:305)
	at org.testng.Assert.assertEquals(Assert.java:315)
	at com.focus.testcases.SellerPriceBookTest.checkVerifyingPurchaseVoucherforRateQuantityApplied(SellerPriceBookTest.java:203)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:86)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:643)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:820)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1128)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)
	at org.testng.TestRunner.privateRun(TestRunner.java:782)
	at org.testng.TestRunner.run(TestRunner.java:632)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:366)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:361)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:319)
	at org.testng.SuiteRunner.run(SuiteRunner.java:268)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1244)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1169)
	at org.testng.TestNG.run(TestNG.java:1064)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </failure>
  </testcase> <!-- checkVerifyingPurchaseVoucherforRateQuantityApplied -->
  <testcase name="checkPurchaseVoucherforLoadingRateforStateApplied" time="38.470" classname="com.focus.testcases.SellerPriceBookTest"/>
  <testcase name="checkSavingItemGroupalongwithItemsinItemMaster" time="33.987" classname="com.focus.testcases.SellerPriceBookTest"/>
  <testcase name="checkCreateTwoDiffBuyerPriceBooksatDiffDateswithSameItem" time="32.578" classname="com.focus.testcases.SellerPriceBookTest"/>
  <testcase name="checkUpdatedBuyerPriceBooksRateValueatVoucherPurchaseVoucher" time="20.284" classname="com.focus.testcases.SellerPriceBookTest"/>
  <testcase name="checkDeletetheStateBuyerPriceBook" time="21.827" classname="com.focus.testcases.SellerPriceBookTest"/>
  <testcase name="checkDeleteSellerPriceBook" time="21.688" classname="com.focus.testcases.SellerPriceBookTest"/>
  <testcase name="checkCreatingBuyerPriceBookforState" time="33.321" classname="com.focus.testcases.SellerPriceBookTest"/>
  <testcase name="checkCreatingTwoPriceBooksforSameItemwithDiffRates" time="20.422" classname="com.focus.testcases.SellerPriceBookTest"/>
  <testcase name="checkDeleteCreatedBuyerPriceBook" time="21.658" classname="com.focus.testcases.SellerPriceBookTest"/>
  <testcase name="checkUpdatingSellerPriceBook" time="6.514" classname="com.focus.testcases.SellerPriceBookTest"/>
  <testcase name="checkCopyPasteAndSavingSellerPriceBook" time="11.071" classname="com.focus.testcases.SellerPriceBookTest"/>
  <testcase name="checkDeletingtheBuyerPriceBook" time="43.249" classname="com.focus.testcases.SellerPriceBookTest"/>
  <testcase name="checkImportingSellerPriceBook" time="38.285" classname="com.focus.testcases.SellerPriceBookTest">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
	at org.testng.Assert.fail(Assert.java:94)
	at org.testng.Assert.failNotEquals(Assert.java:513)
	at org.testng.Assert.assertEqualsImpl(Assert.java:135)
	at org.testng.Assert.assertEquals(Assert.java:116)
	at org.testng.Assert.assertEquals(Assert.java:305)
	at org.testng.Assert.assertEquals(Assert.java:315)
	at com.focus.testcases.SellerPriceBookTest.checkImportingSellerPriceBook(SellerPriceBookTest.java:58)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:86)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:643)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:820)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1128)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)
	at org.testng.TestRunner.privateRun(TestRunner.java:782)
	at org.testng.TestRunner.run(TestRunner.java:632)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:366)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:361)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:319)
	at org.testng.SuiteRunner.run(SuiteRunner.java:268)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1244)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1169)
	at org.testng.TestNG.run(TestNG.java:1064)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </failure>
  </testcase> <!-- checkImportingSellerPriceBook -->
  <testcase name="checkSavedSellerPriceBookatVoucherSalesInvoice" time="55.274" classname="com.focus.testcases.SellerPriceBookTest">
    <error type="org.openqa.selenium.ElementClickInterceptedException" message="element click intercepted: Element &lt;td class=&quot;FGridColumn  &quot; onclick=&quot;FGRIDCONTROL.PRIVATE.onCellClick(this, event);&quot; ondblclick=&quot;FGRIDCONTROL.PRIVATE.onCellDblClick(this, event);&quot; onmouseenter=&quot;null&quot; onmouseleave=&quot;null&quot; style=&quot;border: 1px solid rgb(204, 204, 204); overflow-wrap: break-word; white-space: normal; min-width: 120px; max-width: 120px; text-align: left; padding: 0px;&quot;&gt;&lt;/td&gt; is not clickable at point (231, 522). Other element would receive the click: &lt;div id=&quot;idGlobalError&quot; style=&quot;position: fixed; z-index: 1051; width: 100%; float: right; bottom: 0px;&quot;&gt;...&lt;/div&gt;
  (Session info: chrome=108.0.5359.125)
Build info: version: &#039;3.12.0&#039;, revision: &#039;7c6e0b3&#039;, time: &#039;2018-05-08T14:04:26.12Z&#039;
System info: host: &#039;FOCUS-QA-10&#039;, ip: &#039;192.168.4.206&#039;, os.name: &#039;Windows 10&#039;, os.arch: &#039;amd64&#039;, os.version: &#039;10.0&#039;, java.version: &#039;1.8.0_111&#039;
Driver info: org.openqa.selenium.chrome.ChromeDriver
Capabilities {acceptInsecureCerts: false, browserName: chrome, browserVersion: 108.0.5359.125, chrome: {chromedriverVersion: 108.0.5359.71 (1e0e3868ee06..., userDataDir: C:\Users\Rakesh\AppData\Loc...}, goog:chromeOptions: {debuggerAddress: localhost:51567}, javascriptEnabled: true, networkConnectionEnabled: false, pageLoadStrategy: normal, platform: WINDOWS, platformName: WINDOWS, proxy: Proxy(), setWindowRect: true, strictFileInteractability: false, timeouts: {implicit: 0, pageLoad: 300000, script: 30000}, unhandledPromptBehavior: dismiss and notify, webauthn:extension:credBlob: true, webauthn:extension:largeBlob: true, webauthn:virtualAuthenticators: true}
Session ID: 57c4b0737fc7f18864dbba2c89eb409a">
      <![CDATA[org.openqa.selenium.ElementClickInterceptedException: element click intercepted: Element <td class="FGridColumn  " onclick="FGRIDCONTROL.PRIVATE.onCellClick(this, event);" ondblclick="FGRIDCONTROL.PRIVATE.onCellDblClick(this, event);" onmouseenter="null" onmouseleave="null" style="border: 1px solid rgb(204, 204, 204); overflow-wrap: break-word; white-space: normal; min-width: 120px; max-width: 120px; text-align: left; padding: 0px;"></td> is not clickable at point (231, 522). Other element would receive the click: <div id="idGlobalError" style="position: fixed; z-index: 1051; width: 100%; float: right; bottom: 0px;">...</div>
  (Session info: chrome=108.0.5359.125)
Build info: version: '3.12.0', revision: '7c6e0b3', time: '2018-05-08T14:04:26.12Z'
System info: host: 'FOCUS-QA-10', ip: '192.168.4.206', os.name: 'Windows 10', os.arch: 'amd64', os.version: '10.0', java.version: '1.8.0_111'
Driver info: org.openqa.selenium.chrome.ChromeDriver
Capabilities {acceptInsecureCerts: false, browserName: chrome, browserVersion: 108.0.5359.125, chrome: {chromedriverVersion: 108.0.5359.71 (1e0e3868ee06..., userDataDir: C:\Users\Rakesh\AppData\Loc...}, goog:chromeOptions: {debuggerAddress: localhost:51567}, javascriptEnabled: true, networkConnectionEnabled: false, pageLoadStrategy: normal, platform: WINDOWS, platformName: WINDOWS, proxy: Proxy(), setWindowRect: true, strictFileInteractability: false, timeouts: {implicit: 0, pageLoad: 300000, script: 30000}, unhandledPromptBehavior: dismiss and notify, webauthn:extension:credBlob: true, webauthn:extension:largeBlob: true, webauthn:virtualAuthenticators: true}
Session ID: 57c4b0737fc7f18864dbba2c89eb409a
	at sun.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
	at sun.reflect.NativeConstructorAccessorImpl.newInstance(Unknown Source)
	at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(Unknown Source)
	at java.lang.reflect.Constructor.newInstance(Unknown Source)
	at org.openqa.selenium.remote.http.W3CHttpResponseCodec.createException(W3CHttpResponseCodec.java:187)
	at org.openqa.selenium.remote.http.W3CHttpResponseCodec.decode(W3CHttpResponseCodec.java:122)
	at org.openqa.selenium.remote.http.W3CHttpResponseCodec.decode(W3CHttpResponseCodec.java:49)
	at org.openqa.selenium.remote.HttpCommandExecutor.execute(HttpCommandExecutor.java:158)
	at org.openqa.selenium.remote.service.DriverCommandExecutor.execute(DriverCommandExecutor.java:83)
	at org.openqa.selenium.remote.RemoteWebDriver.execute(RemoteWebDriver.java:543)
	at org.openqa.selenium.remote.RemoteWebElement.execute(RemoteWebElement.java:276)
	at org.openqa.selenium.remote.RemoteWebElement.click(RemoteWebElement.java:83)
	at sun.reflect.GeneratedMethodAccessor3.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.openqa.selenium.support.pagefactory.internal.LocatingElementHandler.invoke(LocatingElementHandler.java:51)
	at com.sun.proxy.$Proxy12.click(Unknown Source)
	at com.focus.Pages.SellerPriceBookPage.checkSavedSellerPriceBookatVoucherSalesInvoice(SellerPriceBookPage.java:12573)
	at com.focus.testcases.SellerPriceBookTest.checkSavedSellerPriceBookatVoucherSalesInvoice(SellerPriceBookTest.java:85)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:86)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:643)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:820)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1128)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)
	at org.testng.TestRunner.privateRun(TestRunner.java:782)
	at org.testng.TestRunner.run(TestRunner.java:632)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:366)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:361)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:319)
	at org.testng.SuiteRunner.run(SuiteRunner.java:268)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1244)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1169)
	at org.testng.TestNG.run(TestNG.java:1064)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </error>
  </testcase> <!-- checkSavedSellerPriceBookatVoucherSalesInvoice -->
  <testcase name="checkPurchaseVoucherRateFieldgivingMinorMaxQuantitygiveninBuyerPriceBook" time="28.027" classname="com.focus.testcases.SellerPriceBookTest">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
	at org.testng.Assert.fail(Assert.java:94)
	at org.testng.Assert.failNotEquals(Assert.java:513)
	at org.testng.Assert.assertEqualsImpl(Assert.java:135)
	at org.testng.Assert.assertEquals(Assert.java:116)
	at org.testng.Assert.assertEquals(Assert.java:305)
	at org.testng.Assert.assertEquals(Assert.java:315)
	at com.focus.testcases.SellerPriceBookTest.checkPurchaseVoucherRateFieldgivingMinorMaxQuantitygiveninBuyerPriceBook(SellerPriceBookTest.java:131)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:86)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:643)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:820)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1128)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)
	at org.testng.TestRunner.privateRun(TestRunner.java:782)
	at org.testng.TestRunner.run(TestRunner.java:632)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:366)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:361)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:319)
	at org.testng.SuiteRunner.run(SuiteRunner.java:268)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1244)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1169)
	at org.testng.TestNG.run(TestNG.java:1064)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </failure>
  </testcase> <!-- checkPurchaseVoucherRateFieldgivingMinorMaxQuantitygiveninBuyerPriceBook -->
  <testcase name="checkRateValueatPurchaseVoucher" time="20.388" classname="com.focus.testcases.SellerPriceBookTest"/>
  <testcase name="checkDeletingSellerPriceBook" time="8.358" classname="com.focus.testcases.SellerPriceBookTest"/>
  <testcase name="checkSavingSellerPriceBookforItemGroupCustomerWisePricebook" time="21.566" classname="com.focus.testcases.SellerPriceBookTest"/>
  <testcase name="checkSavingStatesinStateMaster" time="11.586" classname="com.focus.testcases.SellerPriceBookTest"/>
  <testcase name="checkExportingSellerPriceBook" time="12.602" classname="com.focus.testcases.SellerPriceBookTest"/>
  <testcase name="checkCreatingTwodiffBuyerPricebookforSameProductatDiffDates" time="28.484" classname="com.focus.testcases.SellerPriceBookTest"/>
  <testcase name="checkDeletetheQuantityBuyerPriceBook" time="21.621" classname="com.focus.testcases.SellerPriceBookTest"/>
</testsuite> <!-- com.focus.testcases.SellerPriceBookTest -->
